torch/onnx/_internal/exporter.py

```diff
@@ -633,6 +633,32 @@ def __init__(
         self._fake_context = fake_context
         self._export_exception = export_exception
 
+    def __call__(self, *args: Any, options=None, **kwargs: Any) -> Any:
+        """Runs the ONNX model using ONNX Runtime
+
+        Args:
+            args: The positional inputs to the model.
+            kwargs: The keyword inputs to the model.
+            options: The options to use for running the model with ONNX Runtime.
```

Comment:
Since this is public api I think we should be careful with introducing arguments. Should we make `options` a dataclass?